braga/README.md

```diff
@@ -0,0 +1,655 @@
+![green-seedling-growing](https://cloud.githubusercontent.com/assets/194400/25162588/cca26648-24bb-11e7-8323-42ab19f68be0.jpg)
+
+
+## Why?
+
+One of the ***biggest*** "***challenges***" we faced in **London**
+with **Founders & Coders** was finding ***premises***. <br />
+We have been paying (_burning money on_) "_commercial rates_"
+**£3600/month** (_including "business rates"_) for **140m<sup>2</sup>** <br />
+@ **14 Palmers Road** since January 2015.
+see:
+[Fixed Overhead Breakdown](https://github.com/dwyl/quietspace/tree/58eb654e522b9974c1f83f12b5e79925d0132f3b#fixed-overheads-costs-we-have-to-cover) <br />
+> The amount we pay for 14 Palmers Road is actually _quite reasonable_
+for London commercial property _especially_<br />
+ when compared to a _desk space_
+at "WeWork" or "SecondHome" start at £350/month for **4m<sup>2</sup>** of space!
+
+Even if (_when_) Founders & Coders (London) _eventually_ gets "_free_"
+premises based on its' charitable status everyone _attending_ will
+still need to pay London rents, transport and living expenses.
+
+### Cost of Living in London
+
+The **rent** for **premises** in London
+is the _**tip** of the **cost iceberg**_! <br />
+By _far_ the biggest "_space_" cost is the ***cumulative rent***
+all the people _attending_ <br />
+Founders & Coders (_and subsequently working for @dwyl_)
+pay to _live_ in the city.
+
++ The _average **rent**_ for a _room_ in London
+is **£630/month** (_including bills_).
++ For those who use _public transport_
+a **travel pass** costs a ***minimum*** of
+[**£120/month**](http://content.tfl.gov.uk/adult-fares-2017.pdf) <br />
++ If people eat _frugally_ by preparing meals in bulk/advance,
+_limit alcohol intake_ and "_eating out_"
+they can survive on **£4/day** for **food** so ***£120/month***.
+
+> The _**bare minimum**_ total living cost in London is ***£870/month***. <br />
+This does not factor the wasted _time_ people have commuting <br />
+to-from Mile End each day (_average 1.5h per day!_)
+
+> TODO: @iteles _please_ Insert **Cost of Living Questionnaire** Data/Summary `here` ... https://github.com/dwyl/phase-two/issues/57
+
+## What?
+
+`Braga House` is a **hub** for creative technology learning,
+healthy and sustainable living.
+
+Our `mission` is to give people a comfortable
+yet **_highly_ cost-effective** place to live <br />
+while they are learning the skills to work in (_or start_)
+the top creative technology companies.
+
+
+### Saving _Everyone_ Money/Time
+
+Setting up `Braga House` will cut down on
+_London_ living expenses by ***Two Thirds*** <br />
+and give _everyone_ ***more time*** to learn, create and enjoy life!
+
+The ***Total Cost*** of living in the `Braga House`
```

Comment:
Perhaps the word "Total" should be removed _completely_ because it could be _misinterpreted_ and people could wind up _complaining_ about how they are spending more than €256 per month because they are buying extra food.
Maybe just "Living in the `Braga House` will cost €256 per month".